### DISCUSSION TITLE
Rather than index.vue, is it possible to use the name of the folder as the file also

### DISCUSSION THREAD

**Discussion by keith-at-wildeye** [posva/unplugin-vue-router#447] [Questions and Help] [answered]
One of the things that gets annoying is all the index.vue  files....   I'd much prefer in folder user, that user.vue  would be equivalent of index.vue

Is this possible to do at the moment?

**Comment by posva** [comment, marked as answer]
you could rewrite the paths with the `beforeWriteFiles` option but I would recommend you not to do so.

I think the feature you are looking for is route groups: https://github.com/posva/unplugin-vue-router/issues/448, it would allow you to name the file `(user).vue` without affecting the route structure
